src/setup.ssr.js

```diff
@@ -1,4 +1,4 @@
-globalThis.location = { hostname: 'localhost' }
+globalThis.location = { hostname: 'localhost', host: '127.0.0.1' }
```

Comment:
why host?

Reply:
A temporary hack. Wait for the Next Crypto provider PR to be merged and this can be removed.

Reply:
plz add comment to the line after.

Reply:
I add a notification in my todo app, let's leave the commits clean at the monent